src/MockHandler.php

```diff
@@ -48,12 +48,12 @@ public function append()
     {
         foreach (func_get_args() as $value) {
```

Comment:
please add a changelog too (nothing to do with this line, just needed to comment somewhere)